exporter/signalfxexporter/internal/apm/tracetracker/tracker.go

```diff
@@ -140,7 +140,7 @@ func (a *ActiveServiceTracker) processEnvironment(res pcommon.Resource, now time
 	// Then, try "environment" attribute (SignalFx schema).
 	// Otherwise, use the same fallback value as set on the backend.
 	var environment string
-	if env, ok := attrs.Get(string(conventions.DeploymentEnvironmentKey)); ok {
+	if env, ok := attrs.Get("deployment.environment"); ok {
```

Comment:
Did this get dropped from semconv altogether?

Reply:
Renamed to deployment.environment.name. Given we are going to get rid of this altogether, I have updated to the string for now to limit the changes.